src/base/bittorrent/session.cpp

```diff
@@ -2739,6 +2741,11 @@ void Session::setPort(const int port)
     {
         m_port = port;
         configureListeningInterface();
+
+        if (isReannounceWhenAddressChanged())
+        {
```

Comment:
Please drop the curly braces if the `if ()` body contains only 1 line. Please check other files too.

Reply:
Fix redundant curly braces.